src/tasks/Opps.js

```diff
@@ -36,9 +62,10 @@ const update = ({key, values}, uid, {Profiles, Opps, Projects}) =>
     Projects.get(opp.projectKey).then(project => [user, opp, project])
   )
   .then(([user, opp, project]) =>
-    (isUser(user,opp.ownerProfileKey) ||
+    checkAndSendAcceptanceEmail(key, values, uid, opp, Engagements, Profiles, Opps, Projects) || // eslint-disable-line max-len
```

Comment:
doesn't putting this here cause it to queue sending the email before we're sure the opp will be updated?